modules/help_editor/jsx/helpEditorForm.js

```diff
@@ -0,0 +1,168 @@
+import swal from 'sweetalert2';
+import React, {useState, useEffect} from 'react';
+import {unmountComponentAtNode, createPortal} from 'react-dom';
+import Help from 'jsx/Help';
+
+/**
+ * Help Editor Form Page.
+ *
+ * Serves as an entry-point to the module, rendering the whole react
+ * component page on load.
+ *
+ * Renders Help Editor form page
+ *
+ * @author LORIS Team
+ * @version 1.0.0
+ */
+
+const HelpEditorForm = (props) => {
+  const [title, setTitle] = useState(props.title ?? '');
+  const [content, setContent] = useState(props.content ?? '');
+
+  const returnLabel = 'Return to ' + props.module;
+  const helpContainers = document.getElementsByClassName('help-container');
+
+  useEffect(() => {
+    Array.from(helpContainers).forEach((helpContainer) =>
+      unmountComponentAtNode(helpContainer)
+    );
+  }, []);
+
+  const helpPreview = [];
+  Array.from(helpContainers).forEach((helpContainer) => {
+    helpPreview.push(createPortal(
+      <Help content={
```

Comment:
I don't see why this should be using fragments instead of encapsulated in a div either. Better yet, why not use props.children instead of a content property?

Reply:
That's a good suggestion , thanks!